public/app/features/alerting/unified/rule-editor/formProcessing.ts

```diff
@@ -64,47 +65,63 @@ export function setInstantOrRange(values: RuleFormValues): RuleFormValues {
   };
 }
 
+/**
+ * A alert rule is "transformable" to a simple condition editor if
+ * 1. we have a single data query
+ * 2. we have _either_
+ *   2.1 a reduce expression (pointing to the data query) _and_ a threshold expression pointing to the reducer
+ *   2.2 a threshold expression pointing to a (instant) data query
+ * ⚠️ do not assert on refIds or indexes of the queries
+ */
 export function areQueriesTransformableToSimpleCondition(
-  dataQueries: Array<AlertQuery<AlertDataQuery | ExpressionQuery>>,
+  dataQueries: Array<AlertQuery<AlertDataQuery>>,
   expressionQueries: Array<AlertQuery<ExpressionQuery>>
 ) {
+  // 1. check if we only have a _single_ data query
   if (dataQueries.length !== 1) {
     return false;
   }
-  const singleReduceExpressionInInstantQuery =
-    'instant' in dataQueries[0].model && dataQueries[0].model.instant && expressionQueries.length === 1;
 
-  if (expressionQueries.length !== 2 && !singleReduceExpressionInInstantQuery) {
+  // short-circuit when we have more than 2 expressions, we don't know what to do with that
+  if (expressionQueries.length > 2) {
     return false;
   }
 
-  const query = dataQueries[0];
+  const [dataQuery] = dataQueries;
```

Comment:
Can we change to `dataQueries.at(0)`? I'm not 100% sure, but if the `dataQueries` array is empty then, `dataQuery` variable will be undefined, which might not be properly caught by Typescript